test/wdio/key-reorder/cmp.test.tsx

```diff
@@ -31,6 +31,7 @@ describe('key-reorder', function () {
 
     const button = root.$('button');
     await button.click();
+    await $('#item-4').waitForExist();
```

Comment:
This caused my test to fail locally, so thought of adding this to make it less flaky.

Reply:
Can we note this in a git commit message please?

Reply:
Add comment in the git message